code/__DEFINES/traits.dm

```diff
@@ -70,6 +70,7 @@
 #define	TRAIT_PHOTOGRAPHER		"photographer"
 #define	TRAIT_MUSICIAN			"musician"
 #define	TRAIT_CROCRIN_IMMUNE    "crocin_immune"
+#define TRAIT_NYMPHO			"nymphomania"
```

Comment:
unused trait? you havent made anything in the code use this

Reply:
Couldn't figure out how to force orgasm with nymphos only without adding it in the define.
![](http://puu.sh/DtoA4/b45c7d22a6.png)